.projenrc.js

```diff
@@ -2,7 +2,7 @@ const { awscdk } = require('projen');
 const project = new awscdk.AwsCdkConstructLibrary({
   author: 'JetBridge',
   authorAddress: 'mischa@jetbridge.com',
-  cdkVersion: '2.62.2',
+  cdkVersion: '2.64.0',
```

Comment:
Should this match the latest version of cdk? The later version has the "all headers except host" policy.